tests/test_io/good/output/condition_only_input.catala_en.B.Dcalc

```diff
@@ -1,7 +1,6 @@
 let B =
   λ (B_in_31: B_in{}) →
-    let a.x_32 : bool = error_empty
-      ⟨true ⊢ ⟨⟨true ⊢ false⟩ | false ⊢ ∅ ⟩⟩ in
+    let a.x_32 : bool = error_empty false in
```

Comment:
`error_empty <a non-empty literal>` should be simplified in `<a non-empty literal>` in `dcalc/optimizations.ml`.

Reply:
Maybe it's already the case?